Automattic/sensei
# Add student progress tables based repository

Fixes #5406

### Changes proposed in this Pull Request

* Add custom tables based repositories for course progress, lesson progress and quiz progress.
* Add aggregate repositories that use both types of repositories (comments based and tables based).

### Testing instructions

* Try to take a course with lessons and quizzes.
* Check the `{wp_prefix}sensei_lms_progress` table if it has corresponding entries.
